.github/CODEOWNERS

```diff
@@ -1613,7 +1613,7 @@ sdk/ai/ai-projects @ganeshyb @glharper @dargilco @jhakulin @ZachhK
 /common/tools/ @Azure/azure-sdk-js-dev
 
 # PRLabel: %dev-tool
-/common/tools/dev-tool/ @jeremymeng @mpodwysocki @timovv
+/common/tools/dev-tool/ @jeremymeng @mpodwysocki @timovv @maorleger
```

Comment:
```suggestion
/common/tools/dev-tool/ @jeremymeng @timovv @maorleger
```

Reply:
😢

Reply:
I plan to do a cleanup at some time later.